commands.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+echo "creating elastalert indices"
+python -m elastalert.create_index --config /data/elastalert/config.yaml --verbose
```

Comment:
--verbose should be removed from elastalert-create-index ?
verbose argument is not supported in old elastalert and throws an error. Is it working for new elastalert